### PR TITLE
[PBE-3844] Feature/polls attachment view

### PR DESCRIPTION
**Pull request by JcMinarro**
### 🎯 Goal
Create a new view to render a poll attachment.
Complete: https://stream-io.atlassian.net/browse/PBE-3844?atlOrigin=eyJpIjoiODNlM2IxOGYwZGQ3NGZlMjhkMjA5NmFmODU0YzUyYWEiLCJwIjoiaiJ9
Depends on: #5283

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/internal/PollView.kt [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                onClosePollClick = { onClosePollClick(poll) },
+                onViewPollResultsClick = { onViewPollResultsClick(poll) },
+            )
+            adapter = pollAdapter
+        }
+
+        val pollItems = mutableListOf<PollItem>()
+        pollItems.add(PollItem.Header(poll.name, poll.description))
+        println("JcLog [setPoll]: poll.voteCountsByOption -> ${poll.voteCountsByOption}")
+        println("JcLog [setPoll]: poll.votes.count -> ${poll.votes.size}")
```
Comment (skydoves) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/internal/PollView.kt:86]: Maybe we need to use the logger here or remove it?

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/internal/PollView.kt [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+                        ?: emptyList(),
+                    voteCount = poll.voteCountsByOption[option.id] ?: 0,
+                    isVotedByUser = poll.ownVotes.any { it.optionId == option.id },
+                    totalVotes = poll.voteCountsByOption.values.sum(),
+                    closed = poll.closed,
+                    isWinner = poll.voteCountsByOption[option.id] == poll.voteCountsByOption.values.maxOrNull(),
+                )
+            },
+        )
+        PollItem.ViewResults.takeUnless { poll.ownVotes.isEmpty() }
```
Comment (skydoves) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/internal/PollView.kt:102]: It seems to miss something here to do after getting the filtered list.

**Review by skydoves — APPROVED** [review]
Overall looks good to me. I left some trivial comments :)

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/internal/PollView.kt (hunk 1 above)
Comment (JcMinarro) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/adapter/view/internal/PollView.kt:86]: Good catch.
They were only debug logs during development that are not needed at all

**Pull request merged**